### PR TITLE
Documentation changes for 10.16

### PR DESCRIPTION
**Pull request by Vagdevi1998**

### CODE REVIEW

File: content/dtm/asset-hierarchy-bundle/viewing-assets.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ All subassets and devices for this asset will get listed in the **Subassets** se
 
 5. All the custom properties for this asset will be displayed on the right.
 
+{{< c8y-admon-info>}}
+[Import assets](/dtm/asset-hierarchy/#creating-assets-bulk-import), [Add child assets](/dtm/asset-hierarchy/#add-child-assets) buttons are not visible for the last hierarchical level.<br>
+If the custom property or asset type associated with the selected asset is modified, you can see a warning icon beside the name of the asset type(See Image below).<br>
+You see a warning message if one or more properties assocaited with the asset is not present.
```
Comment (MWindel) [inline content/dtm/asset-hierarchy-bundle/viewing-assets.md:27]:
```suggestion
You see a warning message if one or more properties associated with the asset are not present.
```

File: content/dtm/asset-hierarchy-bundle/viewing-assets.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ All subassets and devices for this asset will get listed in the **Subassets** se
...
+{{< /c8y-admon-info>}}
+
+![Asset Type Modification Warning](/images/dtm/assets/asset-type-modification-warning.png)
```
Comment (MWindel) [inline content/dtm/asset-hierarchy-bundle/viewing-assets.md:30]: Remove the screenshot

**Comment by MWindel** [comment]
Delete the screenshot. It does not contribute anything more to the documentation than is already described in the context.

File: content/dtm/asset-types-bundle/property-library.md [hunk 3]
```diff
@@ -203,7 +204,7 @@ If you create a custom property called "Whiteboard" for a building which has roo
 
 This option allows you to upload a file to define the custom property.
 
-If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. When you create an asset with this custom property, you can only upload files of the allowed file types.
+If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. Provide the maximum file size for uploads **Max file size (MB)** field. When you create an asset with this custom property, you can only upload files of the allowed file types with a file size not exceeding the file size limit.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/property-library.md:207]:
```suggestion
If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. Provide the maximum file size for uploads in the **Max file size (MB)** field. When you create an asset with this custom property, you can only upload files of the allowed file types with a file size not exceeding the file size limit.
```

File: content/dtm/asset-types-bundle/property-library.md [hunk 4]
```diff
@@ -231,7 +233,8 @@ All custom properties matching the search values are displayed on the screen on
 <a name=""></a>
 ### To delete a custom property
 
-Select the respective custom property in the **Property Library** and click **Delete** on the bottom right.
+1. Select the respective custom property in the **Property Library** and click **Delete** on the bottom right.
+2. In the resulting confirmation dialogue, click **Confirm** to continue the deletion or **Cancel** to cancel the deletion.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/property-library.md:237]:
```suggestion
2. In the resulting confirmation dialogue, click **Confirm** to continue.

**Review by MWindel — APPROVED** [review]

File: content/dtm/asset-types-bundle/property-library.md [hunk 5]
```diff
@@ -203,7 +204,7 @@ If you create a custom property called "Whiteboard" for a building which has roo
 
 This option allows you to upload a file to define the custom property.
 
-If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. When you create an asset with this custom property, you can only upload files of the allowed file types.
+If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. Provide the maximum file size for uploads in the **Max file size (MB)** field. When you create an asset with this custom property, you can only upload files of the allowed file types with a file size not exceeding the file size limit.
```
Comment (spau6) [inline content/dtm/asset-types-bundle/property-library.md:207]: We will display either "Max file size in MB" or "Max file size in MB ( Size limit on tenant: 1000 MB)"
Here 1000 MB is a dynamic value (file size for that tenant)
Please consider this change as part of the documentation

Comment (Vagdevi1998) [reply content/dtm/asset-types-bundle/property-library.md]: Modified the field name as per the implementation

**Review by spau6 — APPROVED** [review]

File: content/dtm/asset-types-bundle/property-library.md [hunk 6]
```diff
@@ -203,7 +204,15 @@ If you create a custom property called "Whiteboard" for a building which has roo
 
 This option allows you to upload a file to define the custom property.
 
-If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. When you create an asset with this custom property, you can only upload files of the allowed file types.
+If selected, enter the allowed file types separated by a comma in the **Allowed file types** field. Provide the maximum file size for uploads in the **Max file size in MB** field. When you create an asset with this custom property, you can only upload files of the allowed file types with a file size not exceeding the file size limit. 
+
+{{< c8y-admon-info>}}
+
+If there is a file size limit set on the tenat, you see the limit in the label. For example: If the max file size set on tenant is 1 GB,the label looks like Max file size in MB ( Size limit on tenant: 1024 MB).<br>
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/property-library.md:211]:
```suggestion
If there is a file size limit set for the tenat, you see the limit in the label. 
So if the maximum file size set for the tenant is 1 GB, the label states the maximum file size in MB, for example, 1024 MB.<br>
```

**Comment by MWindel** [comment]
@Vagdevi1998 please incorporate the requested changes before merging.

File: content/dtm/asset-types-bundle/property-library.md (hunk 6 above)
Comment (Vagdevi1998) [reply content/dtm/asset-types-bundle/property-library.md]: The field name will change as per the tenant options. Here Max file size in MB (size limit on tenant: 1024 MB) is the name of the field.

File: content/dtm/asset-types-bundle/asset-types.md [hunk 7]
```diff
@@ -160,6 +160,7 @@ This approach, allows you to create the desired hierarchy including all asset ty
 1. Click the edit icon on the right side of each asset type.
 2. Make your changes in the editor.
 3. Click **Save**.
+4. If this asset type is used by any asset, select **Confirm** in the resulting confirmation dialogue to continue saving your changes.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-types.md:163]: Please change the spelling of dialog

File: content/dtm/asset-types-bundle/property-library.md [hunk 8]
```diff
@@ -216,6 +227,7 @@ To upload the blueprint for a building, you must name the custom property "Build
 1. Select the desired custom property in the **Property Library** to see all the details on the right of the screen.
 2. Edit any content of the custom property.
 3. Click **Save** to save your changes.
+4. If this property is used by any asset type, in the resulting confirmation dialogue, select **Confirm** to continue to save your changes.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/property-library.md:230]: confirmation <s>dialogue</s> dialog

File: content/dtm/asset-types-bundle/property-library.md [hunk 9]
```diff
@@ -233,9 +245,11 @@ All custom properties matching the search values are displayed on the screen on
 
 Select the respective custom property in the **Property Library** and click **Delete** on the bottom right.
 
+In the resulting confirmation dialogue, click **Confirm** to continue.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/property-library.md:248]: confirmation <s>dialogue</s> dialog

File: content/dtm/asset-types-bundle/property-library.md [hunk 10]
```diff
@@ -233,9 +245,11 @@ All custom properties matching the search values are displayed on the screen on
...
+
 {{< c8y-admon-info>}}
 
-If the custom property you want to delete, is part of an existing asset, you see a pop up notification and custom property is not deleted.
+If the custom property you want to delete, is part of an existing asset, you see a error notification and the custom property is not deleted.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/property-library.md:252]:
```suggestion
If the custom property you want to delete is part of an existing asset, you see a error notification and the custom property is not deleted.
```

**Review by lra-sag — CHANGES_REQUESTED** [review]

**Review by lra-sag — APPROVED** [review]

File: content/dtm/asset-hierarchy-bundle/viewing-assets.md [hunk 11]
```diff
@@ -52,5 +57,5 @@ To delete a subasset or device in the hierarchy:
 The delete icon appears on hover of each row under the subassets section.
     {{< /c8y-admon-info>}}
 
-2. In the resulting confirmation dialogue, click **Confirm** to continue or **Cancel** to cancel the deletion.
-3. Optionally select the checkbox in the confirmation dialogue to delete all the subassets and devices for the selected asset.
+2. In the resulting confirmation dialog, click **Confirm** to continue or **Cancel** to cancel the deletion.
```
Comment (MWindel) [inline content/dtm/asset-hierarchy-bundle/viewing-assets.md:60]:
```suggestion
In the following dialog box, click **Confirm** to continue.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**